app-admin/pinkybar/metadata.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+	</maintainer>
+	<maintainer type="person">
+		<email>sacewe@emailure.net</email>
```

Comment:
If you don't have Gentoo Bugzilla account, please create one. If you do have one, please use the same e-mail address here. We use this to assign bugs to you.